backend/tests/users/fixtures.py

```diff
@@ -24,7 +23,7 @@ def fixture_executor_name(monkeypatch: pytest.MonkeyPatch) -> str:
     async def bearer_passthrough(self, request: fastapi.Request):
         return name
 
-    monkeypatch.setattr(JWTBearer, "__call__", bearer_passthrough)
+    monkeypatch.setattr(JWTAPIKeyCookie, "__call__", bearer_passthrough)
```

Comment:
bearer_passthrough is not the right name anymore. Better authentication_passthrough

Reply:
I have renamed it to `cookie_auth_passthrough` as it is still specific to cookies.